server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -762,6 +764,17 @@ public void execute(final String[] args) {
       if (opts.resetSecurity) {
         AccumuloServerContext context = new AccumuloServerContext(new ServerConfigurationFactory(HdfsZooInstance.getInstance()));
         if (isInitialized(fs)) {
+          if (!opts.forceResetSecurity) {
+            String secret = acuConf.get(Property.INSTANCE_SECRET);
+            ConsoleReader c = getConsoleReader();
+            String userEnteredSecret = c.readLine("WARNING: This will remove all users from Accumulo! If you wish to proceed enter the instance secret: ", '*');
+            if (userEnteredSecret != null && !secret.equals(userEnteredSecret)) {
+              log.info("Aborted reset security: Instance secret did not match what is in accumulo-site.xml");
+              return;
+            }
+            log.info("Beginning user initiated reset security on accumulo.");
```

Comment:
Maybe instead of "Beginning user initiated reset security on accumulo" just "Resetting security on Accumulo".